x-pack/plugins/apm/public/components/app/ServiceMap/use_cytoscape_event_handlers.ts

```diff
@@ -133,7 +133,7 @@ export function useCytoscapeEventHandlers({
     };
     const selectHandler: cytoscape.EventHandler = (event) => {
       trackApmEvent({ metric: 'service_map_node_select' });
-      resetConnectedEdgeStyle(event.target);
+      resetConnectedEdgeStyle(event.cy, event.target);
```

Comment:
Shouldn't typescript have caught this?

Reply:
It didn't catch it because event.target is defined as the type `any` (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cytoscape/index.d.ts#L4446), and the second param of `resetConnectedEdgeStyle` is optional.